aderyn_driver/src/driver.rs

```diff
@@ -115,6 +115,11 @@ fn make_context(args: &Args) -> WorkspaceContextWrapper {
     let mut contexts: Vec<WorkspaceContext> =
         process_auto::with_project_root_at(&root_path, &src, &exclude, &remappings, &include);
 
+    if contexts.iter().all(|c| c.src_filepaths.is_empty()) {
+        eprintln!("No solidity files found in given scope!");
```

Comment:
Can you introduce a test for this please @TilakMaddy ?

Reply:
What might we test for in this case ?

Say we form a contexts vector from a directory of solidity files but with no solidity files matching the given scope.

Like, `let mut contexts: Vec<WorkspaceContext> =
        process_auto::with_project_root_at(&root_path, &src, &exclude, &remappings, &include);`

Then test for `contexts.iter().all(|c| c.src_filepaths.is_empty()) == true`  is alright?

Reply:
DONE ✅